dialects/postgresql/src/main/kotlin/app/cash/sqldelight/dialects/postgresql/PostgreSqlTypeResolver.kt

```diff
@@ -92,13 +95,34 @@ class PostgreSqlTypeResolver(private val parentResolver: TypeResolver) : TypeRes
   }
 
   override fun queryWithResults(sqlStmt: SqlStmt): QueryWithResults? {
+    fun List<QueryElement.QueryColumn>.flattenCompounded(): List<QueryElement.QueryColumn> {
```

Comment:
This `fun` is copied from `SelectQueryable`.

Reply:
I guess, all new code was actually copied? What about publishing the code instead and reuse it instead copying it twice (and having it 3 times now)?